Index: b/src/server/storage/mysql-global.conf
===================================================================
--- a/src/server/storage/mysql-global.conf
+++ b/src/server/storage/mysql-global.conf
@@ -100,5 +100,9 @@ wait_timeout=31536000
 # We use InnoDB, so don't let MyISAM eat up memory
 key_buffer_size=16K
 
+# KUBUNTU:
+# Unset the export dir check as only the full mysql-server package creates it
+secure_file_priv=
+
 [client]
 default-character-set=utf8
