From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 11:36:15 +0200
Subject: postgresql9.4_9.5

---
 src/server/storage/dbconfigpostgresql.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/server/storage/dbconfigpostgresql.cpp b/src/server/storage/dbconfigpostgresql.cpp
index 4fe46cf..ee46b9b 100644
--- a/src/server/storage/dbconfigpostgresql.cpp
+++ b/src/server/storage/dbconfigpostgresql.cpp
@@ -85,7 +85,9 @@ bool DbConfigPostgresql::init(QSettings &settings)
                            << QStringLiteral("/usr/lib/postgresql/9.0/bin")
                            << QStringLiteral("/usr/lib/postgresql/9.1/bin")
                            << QStringLiteral("/usr/lib/postgresql/9.2/bin")
-                           << QStringLiteral("/usr/lib/postgresql/9.3/bin");
+                           << QStringLiteral("/usr/lib/postgresql/9.3/bin")
+                           << QStringLiteral("/usr/lib/postgresql/9.4/bin")
+                           << QStringLiteral("/usr/lib/postgresql/9.5/bin");
 
         defaultServerPath = XdgBaseDirs::findExecutableFile(QStringLiteral("pg_ctl"), postgresSearchPath);
         defaultInitDbPath = XdgBaseDirs::findExecutableFile(QStringLiteral("initdb"), postgresSearchPath);
