Last-Update: 2015-07-14
Forwarded: no
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=791807
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: initialise PSQL database with "--data-checksums".

Index: akonadi/src/server/storage/dbconfigpostgresql.cpp
===================================================================
--- akonadi.orig/src/server/storage/dbconfigpostgresql.cpp	2016-01-27 13:48:55.206936407 +0100
+++ akonadi/src/server/storage/dbconfigpostgresql.cpp	2016-01-27 13:51:15.281343111 +0100
@@ -224,6 +224,7 @@
         QStringList arguments;
         arguments << QStringLiteral("--pgdata=%2").arg(mPgData)
                   // TODO check locale
+                  << QStringLiteral("--data-checksums")
                   << QStringLiteral("--locale=en_US.UTF-8");
         QProcess::execute(command, arguments);
     }
